content/en/docs/tasks/administer-cluster/dns-custom-nameservers.md

```diff
@@ -105,34 +105,35 @@ The Corefile configuration includes the following [plugins](https://coredns.io/p
-  The `pods insecure` option is provided for backward compatibility with _kube-dns_. You can use the `pods verified` option, which returns an A record only if there exists a pod in same namespace with matching IP. The `pods disabled` option can be used if you don't use pod records.
+  The `pods insecure` option is provided for backward compatibility with `kube-dns`. You can use the `pods verified` option, which returns an A record only if there exists a pod in same namespace with matching IP. The `pods disabled` option can be used if you don't use pod records.
 * [prometheus](https://coredns.io/plugins/metrics/): Metrics of CoreDNS are available at `http://localhost:9153/metrics` in [Prometheus](https://prometheus.io/) format (also known as OpenMetrics).
-* [forward](https://coredns.io/plugins/forward/): Any queries that are not within the cluster domain of Kubernetes will be forwarded to predefined resolvers (/etc/resolv.conf).
+* [forward](https://coredns.io/plugins/forward/): Any queries that are not within the cluster domain of Kubernetes will be forwarded to predefined resolvers (`/etc/resolv.conf`).
 * [cache](https://coredns.io/plugins/cache/): This enables a frontend cache.
 * [loop](https://coredns.io/plugins/loop/): Detects simple forwarding loops and halts the CoreDNS process if a loop is found.
 * [reload](https://coredns.io/plugins/reload): Allows automatic reload of a changed Corefile. After you edit the ConfigMap configuration, allow two minutes for your changes to take effect.
 * [loadbalance](https://coredns.io/plugins/loadbalance): This is a round-robin DNS loadbalancer that randomizes the order of A, AAAA, and MX records in the answer.
 
 You can modify the default CoreDNS behavior by modifying the ConfigMap.
 
-### Configuration of Stub-domain and upstream nameserver using CoreDNS
+### Configuration of `stubDomains` and `upstreamNameservers` using CoreDNS
 
-CoreDNS has the ability to configure stubdomains and upstream nameservers using the [forward plugin](https://coredns.io/plugins/forward/).
+CoreDNS has the ability to configure `stubDomains` and `upstreamNameservers` using the [forward plugin](https://coredns.io/plugins/forward/).
 
 #### Example
-If a cluster operator has a [Consul](https://www.consul.io/) domain server located at 10.150.0.1, and all Consul names have the suffix .consul.local. To configure it in CoreDNS, the cluster administrator creates the following stanza in the CoreDNS ConfigMap.
+If a cluster operator has a [Consul](https://www.consul.io/) domain server located at `10.150.0.1`, and all Consul names have the suffix `.consul.local`. 
+To configure it in CoreDNS, the cluster administrator creates the following stanza in the CoreDNS ConfigMap.
 
-```
+```yaml
```

Comment:
Not sure json or yaml here.

Reply:
Don't add 'yaml' here. If you are compelled to use a syntax, then use 'none' is okay.